tests/upgrade/post/go.mod

```diff
@@ -0,0 +1,202 @@
+module github.com/oasisprotocol/oasis-core/test-upgrade
+
+go 1.19
+
+replace (
+	// Updates the version used by badgerdb, because some of the Go
+	// module caches apparently have a messed up copy that causes
+	// build failures.
+	// https://github.com/google/flatbuffers/issues/6466
+	github.com/google/flatbuffers => github.com/google/flatbuffers v1.12.1
```

Comment:
Is this still needed?

Reply:
No, the badgerdb uses the same version.

See https://github.com/dgraph-io/badger/blob/v3.2103.4/go.mod#L14.